packages/koa-bolt/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@slack-wrench/koa-bolt",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

```suggestion
  "version": "1.2.0",
```

😅 Sorry, I meant update the [`devDependancy` of `@slack/bolt`](https://github.com/IBM/slack-wrench/pull/101/files#diff-fcf87f9b4ff934fbb9a332b6395310282b96d926e8182986f2c31c374c9ba40fR28) (line 28) so the test cases run against the newer version.

I'll handle bumping this and rolling a release for you.

---

Woops, that makes a lot more sense!